engine/src/flutter/lib/snapshot/BUILD.gn

```diff
@@ -212,36 +209,42 @@ if (host_os == "mac" && (target_os == "mac" || target_os == "ios")) {
       args = [
         rebase_path("${output_dir}/${gen_snapshot_target_name}"),
         rebase_path(
-            "${root_out_dir}/artifacts_$host_cpu/gen_snapshot_${target_cpu}"),
+            "${root_out_dir}/artifacts_$host_cpu/gen_snapshot${gen_snapshot_suffix}"),
```

Comment:
Just checking, is it intentional for this to now resolve to `artifacts_arm/gen_snapshot_armv7` when the `target_cpu` is arm?

Looks like previously only `create_arm_gen_snapshot` task swapped `arm` to `armv7`

Reply:
Yep this is correct. Reason is that this target was never used for 32-bit targets until now, so this hand't been a problem. Originally, it was just used for macOS (arm64, x64), then I later added iOS (arm64 only), but now that it's also used for 32-bit arm target CPUs, this needs to handle armv7 in addition to the other two.

Reply:
Worth noting that I *could* probably remove the `target_os` subexpression from the conditional, but it does help make clear to readers exactly which target platforms we're supporting here and add a bit of future-proofing in case we one day support cross-compile to 32-bit Raspberry Pis from macOS hosts or something 😛.